src/os/ui/layer/vectorlayerui.js

```diff
@@ -1061,8 +1142,18 @@ os.ui.layer.VectorLayerUICtrl.prototype.getShape = function() {
  * @return {Array<string>} The available shape options
  */
 os.ui.layer.VectorLayerUICtrl.prototype.getShapes = function() {
+  return goog.object.getKeys(os.style.SHAPES);
```

Comment:
I've seen this line of code a few places. Would it be worth adding an `os.style.getShapes()` or `os.style.getShapesKeys()` function that everyone calls instead?